bin/phpunit.sh

```diff
@@ -4,4 +4,21 @@ if [[ $RUN_PHPCS == 1 || $SHOULD_DEPLOY == 1 ]]; then
 	exit
 fi
 
-if [ -f "phpunit.phar" ]; then php phpunit.phar -c phpunit.xml.dist $@; else ./vendor/bin/phpunit $@; fi;
+CURRENT_PHP_MAJOR_VERSION=$(php -r 'echo PHP_MAJOR_VERSION;')
+CURRENT_PHP_MINOR_VERSION=$(php -r 'echo PHP_MINOR_VERSION;')
+
+# The PHPUnit version inside composer.json is not compatible with PHP 7.2 and above
+# Update this constant if you wish to bump supported PHP major version
+SUPPORTED_PHP_MAJOR_VERSION_FOR_PHPUNIT_INSTALLED_VIA_COMPOSER_JSON=7
+SUPPORTED_PHP_MINOR_VERSION_FOR_PHPUNIT_INSTALLED_VIA_COMPOSER_JSON=3
+
+if [ $CURRENT_PHP_MAJOR_VERSION -gt $SUPPORTED_PHP_MAJOR_VERSION_FOR_PHPUNIT_INSTALLED_VIA_COMPOSER_JSON ]; then
+	./vendor/bin/phpunit $@;
```

Comment:
`$@` should be bound in double quotes `"$@"`? PHPStorm gives this reference https://github.com/koalaman/shellcheck/wiki/SC2068 

Nitpick: also specifying the configuration file may be better. 


```suggestion
	./vendor/bin/phpunit -c phpunit.xml.dist "$@";
```